source/brailleDisplayDrivers/baum.py

```diff
@@ -18,21 +21,21 @@
 TIMEOUT = 0.2
 BAUD_RATE = 19200
 
-ESCAPE = "\x1b"
-
-BAUM_DISPLAY_DATA = "\x01"
-BAUM_CELL_COUNT = "\x01"
-BAUM_REQUEST_INFO = "\x02"
-BAUM_PROTOCOL_ONOFF = "\x15"
-BAUM_COMMUNICATION_CHANNEL = "\x16"
-BAUM_POWERDOWN = "\x17"
-BAUM_ROUTING_KEYS = "\x22"
-BAUM_DISPLAY_KEYS = "\x24"
-BAUM_ROUTING_KEY = "\x27"
-BAUM_BRAILLE_KEYS = "\x33"
-BAUM_JOYSTICK_KEYS = "\x34"
-BAUM_DEVICE_ID = "\x84"
-BAUM_SERIAL_NUMBER = "\x8A"
+ESCAPE = b"\x1b"

```

Comment:
I think that many drivers would benefit from a new module (e.g. hwIo.constants) containing stuff like ESCAPE, STX, ETX, etc.